apps/mull-api/src/app/app.module.ts

```diff
@@ -47,6 +50,14 @@ import { environment } from '../environments/environment';
         maxFileSize: 10000000000000,
       },
     }),
+    SessionModule.forRoot({
+      session: {
+        name: 'mull-app-session',
+        secret: environment.session.secret,
+        resave: false,
+        saveUninitialized: false,
+      },
+    }),
```

Comment:
Twitter login is using OAuth 1.0 which requires sessions, so it's probably going to be used only for that but it can be useful to track the users in the backend if we need to.

Reply:
Since we have sessions already we can use it to manage authenticated state of the user in the login ticket. NestJS docs suggests [JWT or express sessions](https://docs.nestjs.com/security/authentication)